Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -219,8 +223,10 @@ bool PPCSymbolDB::LoadMap(const std::string& filename, bool bad)
   if (!f)
     return false;
 
-  // four columns are used in American Mensa Academy map files and perhaps other games
-  bool four_columns = false;
+  // Two columns are used by Super Smash Bros. Brawl Korean map file
+  // Three columns are commonly used
+  // Four columns are used in American Mensa Academy map files and perhaps other games
+  int columns = 0;
```

Comment:
Maybe `number_of_columns` or `column_count`, for consistency?

Reply:
Done.